packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -211,6 +211,9 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
     /** If true, throw an error when a tombstone data store is used. */
     private readonly throwOnTombstoneUsage: boolean;
 
+    /** A context should only be marked as deleted when its a remote context */
```

Comment:
I would move this comment to the local data store's delete method. The comment here should explain what deleted means.

Reply:
Sounds good